docker/push-proxy/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:16.04
```

Comment:
can we use another image like `alpine`?

@jwilander do you know if exist any external dependencies?

Reply:
None that I'm aware of

Reply:
@cpanato @jwilander I will test with alpine and update ;)

Reply:
I have tested with Alpine and it was causing issues to the existing push-proxy application. Therefore, I moved to using version 18.04 of Ubuntu, which is the new Ubuntu Minimal version and more details can be found here -> https://blog.ubuntu.com/2018/07/09/minimal-ubuntu-released. The total size of this image is 29MB.